NitroxClient/GameLogic/ExosuitModuleEvent.cs

```diff
@@ -0,0 +1,47 @@
+﻿using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Helper;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic
+{
+    public class ExosuitModuleEvent
```

Comment:
Removing the packet sending will also remove the need for this class.

Reply:
Lets keep this for the future module events ie other Arm Actions.